### PR TITLE
Remove unnecessary use of reflection

### PR DESCRIPTION
**Pull request by basil**

### CODE REVIEW

**Comment by basil** [comment]
@alecharp Build is green

File: pom.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <jenkins.version>2.289.1</jenkins.version>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <useBeta>true</useBeta>
```
Comment (alecharp) [inline pom.xml:27]: Is this really required?

**Review by alecharp — COMMENTED** [review]
Everything lines seem to be fine, but the `useBeta` I don't know..

File: pom.xml (hunk 1 above)
Comment (basil) [reply pom.xml:27]: Yes. To see the reason why, try compiling with it removed.

Comment (basil) [reply pom.xml:27]: I nagged Daniel to remove the scary beta warning in https://github.com/jenkinsci/jenkins/pull/4501#issuecomment-1111098405.

**Pull request merged**